lib/options.js

```diff
@@ -0,0 +1,40 @@
+const websiteInput = document.getElementById('website')
+const addButton = document.getElementById('addWebsite')
+const list = document.getElementById('blacklist')
+
+browser.storage.local.get('config').then(result => {
+  addButton.addEventListener('click', addWebsite)
+  websiteInput.addEventListener('keyup', evt => {
+    if (evt.keyCode === 13) { // Enter
+      addWebsite()
+    }
+  })
+
+  for (const url of result.config.blacklist) {
+    addToList(url, false)
+  }
+
+  function addWebsite () {
+    if (websiteInput.validity.valid) {
+      addToList(websiteInput.value.replace('http://', '').replace('https://'), true)
```

Comment:
Normally it's impossible to add an URL with a /, but I could just do `new URL(websiteInput.value).hostname`, it will be safer and easier to understand. :thinking: